app/views/user_credentials/index.html.erb

```diff
@@ -39,6 +39,7 @@
       <div class="btn-group">
         <%= link_to new_user_credential_path, class: "btn btn-default" do %><span class="glyphicon glyphicon-plus"></span> New Credential<% end %>
         <%= link_to user_credentials_path(format: :json), class: "btn btn-default" do %><span class="glyphicon glyphicon-cloud-download"></span> Download Credentials<% end %>
+        <%= form_tag import_user_credentials_path, multipart: true, class: "btn btn-default" do %><span class="glyphicon glyphicon-upload"><%= file_field_tag :file %><%= submit_tag "Upload File"%></span><% end %>
       </div>
```

Comment:
The button here looks off (words spilled over) compared to the other two buttons. Not sure if this is ok?